tests/characters/test_alice_can_grant_and_revoke.py

```diff
@@ -3,62 +3,80 @@
 import maya
 import pytest
 from apistar.test import TestClient
-from bytestring_splitter import BytestringSplitter
-from constant_sorrow import constants
-from umbral.fragments import KFrag
-from umbral.keys import UmbralPublicKey
 
 from nucypher.characters import Ursula
 from nucypher.crypto.api import keccak_digest
 from nucypher.crypto.powers import SigningPower, EncryptingPower
+from umbral.fragments import KFrag
+
+
+class MockPolicyCreation:
+    """
```

Comment:
I actually pulled this out of use at the last minute, but it will be used in LL3.

Reply:
:heavy_check_mark: